meson.build

```diff
@@ -208,10 +208,6 @@ if get_option('icu_builtin')
   conf.set('HAVE_ICU_BUILTIN', 1)
 endif
 
-if get_option('experimental_api')
-  conf.set('HB_EXPERIMENTAL_API', 1)
-endif
```

Comment:
Great that alll are stable now, just make sure https://github.com/google/oss-fuzz/blob/master/projects/harfbuzz/build.sh#L32 will be removed while the merge

Reply:
Thanks for the heads-up. That would be a good problem to have when we merge this. :)